keycloak/keycloak-admin-ui
# Edit message bundle values

## Motivation

https://issues.redhat.com/browse/CLUX-162
Closes #684 

## Verification Steps



1. Go to Realm Settings -> Localization
2. Enable internationalization
3. Save and add a message bundle to the data table below.
4. Click the edit icon to the right of the key/value pair you added.
5. Verify that the value can be edited, updated, and saved correctly.

## Checklist:


## Additional Notes

Tests will be added in separate PR because #1677 must be fixed first
